editor/inspector_dock.cpp

```diff
@@ -541,9 +541,10 @@ void InspectorDock::update(Object *p_object) {
 	p->add_shortcut(ED_SHORTCUT("property_editor/expand_revertable", TTR("Expand Non-Default")), EXPAND_REVERTABLE);
 
 	p->add_separator(TTR("Property Name Style"));
-	p->add_radio_check_item(TTR("Raw"), PROPERTY_NAME_STYLE_RAW);
-	p->add_radio_check_item(TTR("Capitalized"), PROPERTY_NAME_STYLE_CAPITALIZED);
-	p->add_radio_check_item(TTR("Localized"), PROPERTY_NAME_STYLE_LOCALIZED);
+	p->add_radio_check_item(TTR("Raw") + " (e.g. \"z_index\")", PROPERTY_NAME_STYLE_RAW);
+	p->add_radio_check_item(TTR("Capitalized") + " (e.g. \"Z Index\")", PROPERTY_NAME_STYLE_CAPITALIZED);
```

Comment:
should this be
```suggestion
	p->add_radio_check_item(vformat(TTR("Raw (e.g. \"%s\")"), "z_index"), PROPERTY_NAME_STYLE_RAW);
	p->add_radio_check_item(vformat(TTR("Capitalized (e.g. \"%s\")"), "Z Index"), PROPERTY_NAME_STYLE_CAPITALIZED);
```
?
"e.g." makes sense only in English.